backend/spec/requests/tags_spec.rb

```diff
@@ -0,0 +1,145 @@
+# frozen_string_literal: true
+
+RSpec.describe 'Tags' do
+  let!(:tags) do
+    (1..3).map do |index|
+      create(:tag, priority: index)
+    end.index_by(&:priority)
+  end
+
+  describe 'GET /tags' do
+    it '全てのメモが取得でき昇順で並び変えられている' do
+      aggregate_failures do
+        get '/tags'
+        expect(response).to have_http_status(:ok)
+        assert_response_schema_confirm(200)
+        expect(response.parsed_body.length).to eq(3)
+
+        tag_ids = tags.values.map(&:id)
+        response.parsed_body.each do |tag|
+          expect(tag_ids).to include(tag['id'])
+        end
+
+        tag_names = tags.values.map(&:name)
+        response.parsed_body.each do |tag|
+          expect(tag_names).to include(tag['name'])
+        end
+
+        tag_priorities = tags.values.map(&:priority)
+        expect(response.parsed_body.pluck('priority')).to eq(tag_priorities.sort)
+      end
+    end
+  end
+
+  describe 'POST /tags' do
+    context 'タグ名が有効な場合' do
+      let(:valid_tag_params) { { name: 'New Tag', priority: 4 } }
```

Comment:
IMO/Simple

このcontextに閉じたpramaterなので、
valid_tag_paramsという命名ではなく、paramsでいいと思います！
これ以外のテストケースの`invalid_tag_params`も同様にです！
```suggestion
      let(:params) { { name: 'New Tag', priority: 4 } }
```

Reply:
確かに、その方がsimpleで良いですね👍
paramsに修正します！